grakn-graph/src/main/java/ai/grakn/graph/internal/RelationImpl.java

```diff
@@ -234,7 +201,6 @@ void cleanUp() {
      */
     @Override
     public void delete() {
-        scopes().forEach(this::deleteScope);
```

Comment:
Thanks, that's fine! Am I right in thinking that deleting the relation also used to delete the scope? Not that it matters, since we weren't using the old scope behaviour.